app/main.py

```diff
@@ -8,7 +8,7 @@
 from basyx.aas.backend.local_file import LocalFileObjectStore
 from basyx.aas.adapter.http import WSGIApp
 
-storage_path = os.getenv("STORAGE_PATH", "storage")
+storage_path = os.getenv("STORAGE_PATH", "/storage")
```

Comment:
I'm a bit confused as to why this is shown as a change. If you look at the current state of `main`, it should be already `/storage`. Can you make sure your branch is up to date with `main`? (At this step you can also resolve the current conflicts).